.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -25,7 +25,13 @@ Share any alternative solutions or features considered.
 **How does this impact you?**  
 Explain the impact to your work or process, e.g., We cannot do [...].
 
-**Are there any deadlines?**  
+**What are the customer data privacy implications for this request?**
+Will (or could) sensitive customer data be exposed as part of this feature? Please test any suggested commands yourself and list any tokens, keys, PII, or other sensitive information that is at risk of being exported or shared via this feature.
```

Comment:
How will they be able to test suggested commands that don't exist? Can we just ask them to list any tokens, keys, PII, etc?

Reply:
I think the vast majority of new features people have been asking for are either new runners that mirror some existing shell command or web-ui action, or just new (existing) runners passing commands through. I'm happy to re-word this though if you have a suggestion for alternate text you'd prefer!

Reply:
Ok, that's helpful to know. No need to reword. I'll resolve this conversation as is.